cache.js

```diff
@@ -0,0 +1,8 @@
+const redis = require('redis');
+const client = redis.createClient();
```

Comment:
Is the redis client supposedly to be running locally? How would we manage scalability issues?